modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+In some circumstances, this resulted in the subsequently pasted material containing unexpected characters. Specifically, normal spaces were sometimes replaced with non-breaking spaces, `&nbsp;`.
+
+In {productname} 6.1, documents in read-only mode no longer captures copy events. Rather, they white-list copy events and pass them to the appropriate event listener (either {productname}’s built-in capability or the PowerPaste plugin).
+
+Consequently, unexpected characters no longer present in material copied-and-pasted from read-only {productname} documents.
+
+
+=== Ctrl-Shift-Home/End or ⌘-Shift-Up-arrow/Down-arrow did not expand the selection to a `contenteditable="false"` element if the element was at the beginning or end of a document
+
+The keyboard shortcut Ctrl-Shift-Home or ⌘-Shift-Up-arrow selects all material from the insertion point to a document’s beginning. Equivalently, Ctrl-Shift-End or ⌘-Shift-Down-arrow selects all material from the insertion point to a document’s end.
+
+NOTE: These selection behaviors are not {productname}-specific. These keyboard shortcuts select in this fashion in virtually all applications.
+
+However, if the first element in a {productname} document had a `contenteditable="false"` property, the Ctrl-Shift-Home or ⌘-Shift-Up-arrow keystrokes did not extend the selection to the first element. That is, the selection extended to the trailing edge of the element with a `contenteditable="false"` property but no further.
+
+Equivalently, if the last element in a {productname} document had a `contenteditable="false"` property, Ctrl-Shift-End or ⌘-Shift-Down-arrow extended the selection only to the leading edge of this final element.
+
+This selection behavior (ie not allowing the generic keyboard shortcuts to extend the selection to elements with `contenteditable="false"` properties if these elements are at the start of end of the material being selected) is the native behavior of all host browsers supported by {productname}.
+
+As of {productname} 6.1, however, an override has been introduced. This override enables {productname} to extend selection of `contenteditable="false"` elements at the beginning or end of {productname} documents using these generic keyboard shortcuts.
+
+Consequently, Ctrl-Shift-Home or ⌘-Shift-Up-arrow and Ctrl-Shift-End or ⌘-Shift-Down-arrow now works in {productname} documents as they do in almost all other applications.
+
+
+=== `<pre>` tags were not preserved when copying and pasting
+
+Previously, when text set between `<pre>` tags was cut or copied and then pasted to a new line in a {productname} document, the `<pre>` tags were not preserved on pasting.
+
+This update corrects this. `<pre>` formatted text copied or cut will now continue to be `<pre>` formatted text when pasted elsewhere in a {productname} document.
+
+
+=== Preview and Insert Template dialogs now display the correct content background color when using dark skins
```

Comment:
You added the word `themed` in the line below, which I think is the right way to approach this, but I suggest it needs to go in the heading as well :)

```suggestion
=== Preview and Insert Template dialogs now display the correct content background color when using dark themed skins
```